pkg/mounter/mounter-s3fs.go

```diff
@@ -17,22 +17,26 @@ type s3fsMounter struct {
 	endPoint   string //From Secret in SC
 	regnClass  string //From Secret in SC
 	accessKeys string
+	authType string
 }
 
 const (
 	s3fsCmd  = "s3fs"
 	metaRoot = "/var/lib/ibmc-s3fs"
 	passFile = ".passwd-s3fs"
+	// defaultIAMEndPoint is the default URL of the IBM IAM endpoint
+	defaultIAMEndPoint = "https://iam.cloud.ibm.com"
```

Comment:
Get the IAM EP from secret as well.

Reply:
https://github.com/IBM/satellite-object-storage-plugin/blob/93600ef87437ad937486b1253fa8c45ac11ba833/pkg/driver/controllerserver.go#L70

We have option to get IAM Endpoint from secret as well. If not present we fall back to default value.